_includes/app/meta.html

```diff
@@ -4,5 +4,5 @@
 
 {% seo %}
 
-<link rel="icon" href="https://media.prx.org/favicon/icon-32.ico" sizes="any">
-<link rel="icon" href="https://media.prx.org/favicon/icon.svg" type="image/svg+xml">
+<link rel="icon" href="/assets/img/icon-32.ico" sizes="any">
+<link rel="icon" href="/assets/img/icon.svg" type="image/svg+xml">
```

Comment:
These should continue to reference `media.prx.org`. Only [favicon.ico](https://github.com/PRX/annual-report-2021/blob/main/favicon.ico) should be included in the project, and it shouldn't be referenced anywhere in the HTML. This is because some things that don't read/render HTML will still look for that file (and only that file) at the root of the domain.

Reply:
Ah!!!! Got it.

Reply:
pushed up